tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -48,6 +53,60 @@
 export_cmd = "export KUBECONFIG=" + kubeconfig_path
 
 
+def test_remove_rancher_ha():
+    resource_prefix = \
+        CATTLE_TEST_URL.split(".qa.rancher.space")[0].split("//")[1]
+    admin_token = get_user_token("admin", ADMIN_PASSWORD)
+    client = get_client_for_token(admin_token)
+
+    # delete clusters except the local cluster
+    clusters = client.list_cluster(id_ne="local").data
+    print("deleting the following clusters: {}"
+          .format([cluster.name for cluster in clusters]))
+    for cluster in clusters:
+        delete_cluster(client, cluster)
+
+    # delete nodes
+    node_filter = [{
+        'Name': 'tag:Name',
+        'Values': [resource_prefix + "-*"]
+    }]
+    nodes = AmazonWebServices().get_nodes(filters=node_filter)
+    print("deleting the following instances: {}"
+          .format([node.public_ip_address for node in nodes]))
+    AmazonWebServices().delete_nodes(nodes)
```

Comment:
After the node deletion, can we assert the length of nodes list is zero ? 
If the node deletion fails, can we retry after a short time and exit with a proper message ?

Reply:
The best effort we can do is to send out the deletion request to AWS, we don't need to check if the deletion succeeds or not.